deployment/environment.go

```diff
@@ -77,6 +77,7 @@ type Environment struct {
 	NodeIDs           []string
 	Offchain          OffchainClient
 	GetContext        func() context.Context
+	OCRSecrets        OCRSecrets
```

Comment:
I would ensure that we plumb this through the memory tests, or we'll end up with empty OCR secrets in tests. That's fine for most tests, but we will want to pull over changesets from chainlink-deployments which use this, and it will not be possible for them to run in tests, unless we can plumb through fake secrets.  I'd follow the PR where I did GetContext() above for hints as to the entry-points. But in itself, this is fine. Technically it's not YET needed at this step. Just don't let's forget about it.

Reply:
I spent some time on this today, have another look!